### PR TITLE
feat(*): support community elements by using peer dependencies

### PR DESCRIPTION
**Pull request by TremayneChrist**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -30,10 +30,10 @@ jobs:
         run: npm ci --ignore-scripts --audit=false --fund=false
 
       - name: Production build
-        run: lerna run build:prod --include-dependencies --since
+        run: npm run build:prod
 
       - name: Bump version and create GitHub release
-        run: lerna version --create-release github --conventional-commits --yes
+        run: lerna version --create-release github --conventional-commits --conventional-graduate --yes
```
Comment (wsuwt) [inline .github/workflows/release.yml:36]:
```suggestion
        run: lerna version --create-release github --conventional-commits --yes
```

File: documents/src/pages/elements/combo-box.md [hunk 2]
```diff
@@ -346,13 +346,13 @@ comboBox.addEventListener('query-changed', (event) => {
 
 Combo Box supports custom rendering by providing a renderer function to the `renderer` property. The renderer receives a data item, Collection Composer and previously mapped item elements (if any), and must return an `HTMLElement`.
 
-The preferred approach is to extend the `DefaultRender` that comes with Combo Box. The default renderer uses [Item](./elements/item) elements, and supports highlighted, selected, disabled, hidden and readonly states.
+The preferred approach is to extend the `ComboBoxRenderer` that comes with Combo Box. The default renderer uses [Item](./elements/item) elements, and supports highlighted, selected, disabled, hidden and readonly states.
 
 ```javascript
-import { DefaultRenderer } from '../lib/ef-combo-box';
+import { ComboBoxRenderer } from 'ef-combo-box';
```
Comment (wsuwt) [inline documents/src/pages/elements/combo-box.md]:
```suggestion
import { ComboBoxRenderer } from '@refinitiv-ui/elements/combo-box';
```

File: package.json [hunk 3]
```diff
@@ -11,17 +11,20 @@
   },
   "scripts": {
     "lerna": "lerna",
-    "build": "lerna run build",
-    "build:docs": "lerna run --scope @refinitiv-ui/docs build --stream",
-    "build:prod": "lerna run build:prod",
+    "build": "nx run-many --target build --parallel --all",
+    "build:affected": "nx affected:build --parallel",
+    "build:docs": "nx run @refinitiv-ui/docs:build",
+    "build:prod": "nx run-many --target build:prod --all",
```
Comment (wsuwt) [inline package.json:17]:
```suggestion
    "build:prod": "nx run-many --target build:prod --parallel --all",
```

Is it possible to also have `parallel` flag here ?

Comment (wsuwt) [reply package.json:17]: Or maybe we could have a single `parallel` setting in `nx.json` as I see that nx parallel still respects the package dependency graph.

File: package.json [hunk 4]
```diff
@@ -11,17 +11,20 @@
...
     "start": "node cli start",
-    "start:docs": "lerna run --scope @refinitiv-ui/docs start --stream",
+    "start:docs": "nx run @refinitiv-ui/docs:start",
     "test": "node cli test",
```
Comment (wsuwt) [inline package.json:20]: I think adding `reset` command to clear cache would be really useful. 
```suggestion
    "test": "node cli test",
    "reset": "nx reset",
```

File: package.json (hunk 3 above)
Comment (TremayneChrist) [reply package.json:17]: This was removed to have a full log for production builds.

File: .github/workflows/release.yml (hunk 1 above)
Comment (TremayneChrist) [reply .github/workflows/release.yml:36]: I added this to make sure that we would always create a prod version release. e.g. if we ended up merging a prerelease into dev -> main.

**Pull request merged**